libs/src/sdk/oauth/Oauth.ts

```diff
@@ -109,27 +117,25 @@ export class Oauth {
   loginSuccessCallback: LoginSuccessCallback | null
   loginErrorCallback: LoginErrorCallback | null
 
-  constructor(discoveryProvider: DiscoveryProvider) {
+  constructor({ discoveryProvider, appName }: OauthConfig) {
     if (typeof window === 'undefined') {
       // TODO(nkang): Add link to documentation once written
       throw new Error(
         'Audius OAuth SDK functions are only available in browser. Refer to our documentation to learn how to implement Audius OAuth manually.'
       )
     }
     this.discoveryProvider = discoveryProvider
-    this.appName = null
+    this.appName = appName ?? null
```

Comment:
The app name can't be optional because the Oauth popup requires it - can we make it required?

Reply:
Oh yup, will update!